WORKSPACE

```diff
@@ -23,71 +23,75 @@ python_configure(
     name = "local_config_python",
     python_version = "3",  # required to use `python3-config`
 )
-############################# TensorFlow Setup ###############################
+############################# OpenXLA Setup ###############################
 
-# To update TensorFlow to a new revision,
+# To update OpenXLA to a new revision,
 # a) update URL and strip_prefix to the new git commit hash
 # b) get the sha256 hash of the commit by running:
-#    curl -L https://github.com/tensorflow/tensorflow/archive/<git hash>.tar.gz | sha256sum
+#    curl -L https://github.com/openxla/xla/archive/<git hash>.tar.gz | sha256sum
 #    and update the sha256 with the result.
 http_archive(
-    name = "org_tensorflow",
+    name = "xla",
     patch_args = [
         "-l",
         "-p1",
     ],
     patch_tool = "patch",
     patches = [
-        "//tf_patches:absl_statusor.diff",
-        "//tf_patches:cache_urls.diff",
-        "//tf_patches:cuda_graph.diff",
-        "//tf_patches:f16_abi_clang.diff",
-        "//tf_patches:gpu_race_condition.diff",
-        "//tf_patches:grpc_version.diff",
-        "//tf_patches:optimized_function_graph.diff",
-        "//tf_patches:profiler_trace.diff",
-        "//tf_patches:stream_executor.diff",
-        "//tf_patches:thread_local_random.diff",
-        "//tf_patches:topk_rewriter.diff",
-        "//tf_patches:triton_filesystem.diff",
-        "//tf_patches:xla_bzl.diff",
-        "//tf_patches:xplane.diff",
+        "//openxla_patches:absl_statusor.diff",
+        "//openxla_patches:cache_urls.diff",
+        "//openxla_patches:cuda_graph.diff",
+        "//openxla_patches:f16_abi_clang.diff",
+        "//openxla_patches:gpu_race_condition.diff",
+        "//openxla_patches:grpc_version.diff",
+        "//openxla_patches:optimized_function_graph.diff",
+        "//openxla_patches:profiler_trace.diff",
+        "//openxla_patches:stream_executor.diff",
+        "//openxla_patches:thread_local_random.diff",
+        "//openxla_patches:topk_rewriter.diff",
+        "//openxla_patches:triton_filesystem.diff",
+        "//openxla_patches:xla_bzl.diff",
+        "//openxla_patches:xplane.diff",
```

Comment:
do we still need all of these patches after we moved to OpenXLA? I thought a lot of them are tf specified

Reply:
all the patches are applied to OpenXLA according to current TF-pin;

according to our branch `newbranchopxla`:
- `openxla_patches/optimized_function_graph.diff` is empty now, could delete this file
- `openxla_patches/xplane.diff` is empty now, could delete this file

other patch file would need confirmed before delete

Reply:
since https://github.com/pytorch/xla/pull/5219 has passed all tests, will apply this change later